src/router/routes.js

```diff
@@ -1,5 +1,5 @@
 export default [
-	{ path: '/', component: () => import('@/pages/Homepage/Homepage') },
+	{ path: '/', component: () => import('@/pages/Homepage/iwd/IWDHomepage') },
```

Comment:
@mcstover Will need a game plan for this route

Reply:
@ryan-ludwig I think this will be fine for demo and testing tomorrow. We'll likely want to move back to a single page level component for the Homepage, then within that component we'll use a setting to toggle between 2 content components and the theme props.